.bazelrc

```diff
@@ -0,0 +1,29 @@
+#
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# Common flags for Clang
+build:clang --repo_env=CC=clang
+
+# Flags for Clang with ASAN and libfuzzer
+build:asan-libfuzzer --config=clang
+build:asan-libfuzzer --linkopt=-fsanitize=fuzzer,address
+build:asan-libfuzzer --copt=-fsanitize=fuzzer,address
+
+# Flags for Clang with MSAN
+build:msan --config=clang
+build:msan --linkopt=-fsanitize=memory,fuzzer
+build:msan --copt=-fsanitize=memory,fuzzer
+build:msan --copt=-fsanitize-memory-track-origins=2
```

Comment:
@asraa for her input too.

This flag makes the MSAN reports more actionable, but comes at a ~2x slowdown cost. Should we have this on by default for all fuzzer builds?

I'm wondering if we can define a separate `--config=msan-libfuzzer-repro` configuration used for reproducing crashes only.